liblangutil/Scanner.cpp

```diff
@@ -349,18 +413,30 @@ size_t Scanner::scanSingleLineDocComment()
 
 Token Scanner::skipMultiLineComment()
```

Comment:
This PR only deals with comments, but what about string literals?

Also test-wise it would be nice to have tests for using such unicode symbols in identifiers and keywords as well. Those should be flat out rejected, as they expect ASCII only, but having a comprehensive test suite can't hurt.

Reply:
didn't think of string literals, good point, thanks. As of the other parts, I didn't expect that to work due to lexical grammar. I'll add at least string literals to the checklist.

Reply:
> As of the other parts, I didn't expect that to work due to lexical grammar. 

Sure, but adding a simple test case when others are added for directional codes should not hurt.